runelite-client/src/main/java/net/runelite/client/plugins/mta/enchantment/EnchantmentRoom.java

```diff
@@ -143,4 +131,48 @@ public boolean inside()
 		return player != null && player.getWorldLocation().getRegionID() == MTA_ENCHANT_REGION
 			&& player.getWorldLocation().getPlane() == 0;
 	}
+
+	@Override
+	public void under(Graphics2D graphics)
+	{
+		final MarkingMode mode = config.dragonstoneMode();

```

Comment:
Prefer config checks to be after `inside()` as it is much more expensive.

Reply:
Changed with the latest commit

Reply:
Changed with the latest commit